### PR TITLE
Getting started information.

### PR DESCRIPTION
**Pull request by garnertb**
image

SQL to run for existing:
```
alter table "FireDepartments" add column integration_complete BOOLEAN DEFAULT FALSE;
alter table "FireDepartments" add column integration_verified BOOLEAN DEFAULT FALSE;
```

### CODE REVIEW

File: client/app/user/user-home/user-home.controller.js [hunk 1]
```diff
@@ -14,6 +16,31 @@ export default class UserHomeController {
       $('html, body').animate({ scrollTop: $(location).offset().top }, 1000);
     };
 
+    this.gettingStarted = [
+      // { step: 'Assigned Fire Department',
+      //   status: _.isNumber(this.principal.fire_department__id),
+      //   action: 'What department are you with?'
+      // },
+      { step: 'Create an Account',
```
Comment (chopchop505) [inline client/app/user/user-home/user-home.controller.js]: I'm assuming they will always be signed up if they hit this page...but we can keep this here

File: client/app/user/user-home/user-home.controller.js [hunk 2]
```diff
@@ -14,6 +16,31 @@ export default class UserHomeController {
...
+        status: !_.isEmpty(this.principal),
+        action: 'Sign up!'
+      },
+      { step: 'Data Integration',
+        status: _.get(this.fireDepartment, 'integration_complete', false),
+        action: 'Contact our integration team at contact@statengine.io to learn how to integrate your data into StatEngine.'
```
Comment (chopchop505) [inline client/app/user/user-home/user-home.controller.js]: can you add styling to email address?

File: client/app/user/user-home/user-home.controller.js (hunk 1 above)
Comment (chopchop505) [inline client/app/user/user-home/user-home.controller.js]: I'm assuming they will always be signed up if they hit this page...but we can keep this here

**Comment by garnertb** [comment]
Feedback implemented.  This one should be good to go now.

**Pull request merged**